CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ find_package(ament_cmake REQUIRED)
 find_package(rclcpp REQUIRED)
 find_package(geometry_msgs REQUIRED)
 find_package(sensor_msgs REQUIRED)
+find_package(OpenCV REQUIRED)
```

Comment:
```suggestion
find_package(OpenCV COMPONENTS core video imgproc contrib REQUIRED)
```

Reply:
I observed the other error.
```
root@13a54f4a8dca:/src# colcon build
Starting >>> blackchicken
--- stderr: blackchicken                         
CMake Error at /usr/share/cmake-3.10/Modules/FindPackageHandleStandardArgs.cmake:137 (message):
  Could NOT find OpenCV (missing: contrib) (found version "3.2.0")
Call Stack (most recent call first):
  /usr/share/cmake-3.10/Modules/FindPackageHandleStandardArgs.cmake:378 (_FPHSA_FAILURE_MESSAGE)
  /usr/share/OpenCV/OpenCVConfig.cmake:260 (find_package_handle_standard_args)
  CMakeLists.txt:19 (find_package)


make: *** [cmake_check_build_system] Error 1
---
Failed   <<< blackchicken	[ Exited with code 2 ]

Summary: 0 packages finished [0.96s]
  1 package failed: blackchicken
  1 package had stderr output: blackchicken
```

Reply:
This errors looks like missing opencv_contrib  in the docker image.
you have to prepare the docker image fulfills that requirements.